### PR TITLE
SWEN-21

### PR DESCRIPTION
**Pull request by vdd5**

### CODE REVIEW

File: src/test/java/com/sweng894/GetVaccinated/VaccineTests.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package com.sweng894.GetVaccinated;
 
-import com.sweng894.GetVaccinated.model.Vaccine;
+import com.sweng894.GetVaccinated.api.entity.Vaccine;
```
Comment (rsolomo) [inline src/test/java/com/sweng894/GetVaccinated/VaccineTests.java:3]: This is removing test coverage from from the pre-existing model.Vaccine class. We should have a new suite of tests for the new vaccine class or consolidate onto one Vaccine class.

File: src/main/java/com/sweng894/GetVaccinated/api/controller/VaccineController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.sweng894.GetVaccinated.api.controller;
+
+
+import com.sweng894.GetVaccinated.api.entity.Vaccine;
+import com.sweng894.GetVaccinated.api.repository.VaccineRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class VaccineController {
```
Comment (rsolomo) [inline src/main/java/com/sweng894/GetVaccinated/api/controller/VaccineController.java:10]: What is your plan for rendering content on the page? REST-like endpoints wouldn't be needed for this story if server-side rendering is used. Is the intent to client-side render the content?

**Comment by vdd5** [comment]
@rsolomo Please review the code. It's ready to merge. I refactored according to your comments

**Comment by vdd5** [comment]
@rsolomo Resolved all the issues as you said. Please let me know if there's anything else.

**Pull request merged**